src/bilder/components/hashicorp/consul/models.py

```diff
@@ -116,8 +116,12 @@ def render_configuration_files(self) -> Iterable[tuple[Path, str]]:
 
     @property
     def data_directory(self) -> Path:
-        data_dir = next(iter(filter(
-                None,
-                map(lambda config: config.data_dir, self.configuration.values()),
-            )))
+        data_dir = next(
+            iter(
+                filter(
+                    None,
+                    map(lambda config: config.data_dir, self.configuration.values()),
+                )
+            )
+        )
```

Comment:
I don't know what this is. Must be something ruff/black decided to do when I ran pre-commit.